### PR TITLE
SSBFFA: Redroller's custom ability

### PR DESCRIPTION
**Pull request by Volco**

### CODE REVIEW

**Comment by Volco** [comment]
theres only 3 0s thats a "few" not "many" but i'll test it without 2 of them

**Comment by Lycanium-Z** [comment]
Percentages man. But still 2 unnecessary Zeros (fixed)

File: mods/cssb/abilities.js [hunk 1]
```diff
@@ -184,4 +184,31 @@ exports.BattleAbilities = {
...
+					return stat;
+				};
+			},
+			onResidualOrder: 23,
+			onEnd: function () {
+				this.add('-fieldend', 'move: Trick Room');
+				this.getStatCallback = null;
+			},
+		},
+		desc: "On switch-in, user activates permanent trick room, uses will-o-wisp, gains 2x def and spd.",
```
Comment (DeathlyPlays) [inline mods/cssb/abilities.js]: This is very nit-picky, but can you please upper case properly and completely spell Defense and Special Defense (and make a shortDesc for abbreviations (assuming we want to be like main?))

Comment (Volco) [reply mods/cssb/abilities.js]: @DeathlyPlays it doesnt fit if they are spelled out

Comment (Volco) [reply mods/cssb/abilities.js]: and !dt ability name ONLY shows regular desc

**Comment by Volco** [comment]
@DeathlyPlays cant spell out the Def and SpD because it wont fit

**Comment by Lycanium-Z** [comment]
It doesnt matter tbh

**Comment by Volco** [comment]
^

File: mods/cssb/abilities.js (hunk 1 above)
Comment (Lycanium-Z) [reply mods/cssb/abilities.js]: Actually it shows shortdesc if there is one. But desc if no shortdesc

**Comment by DeathlyPlays** [comment]
shortDesc is what shows on /dt unless there isn't one which defaults to desc

**Comment by Volco** [comment]
any last comments before i merge this?

**Comment by WorseThanDeath** [comment]
Everything seems fine to me

**Pull request merged**